proxy.conf.example

```diff
@@ -13,29 +14,54 @@ server {
     proxy_set_header Host $http_host;
     proxy_set_header X-NginX-Proxy true;
 
-    # The root path goes to the PDC API implementation, the web service.
+    # The PDC API implementation, the web service.
     location / {
         proxy_pass http://web:3000;
     }
+}
+
+server {
+    listen 0.0.0.0:8443 ssl;
+    server_name # Add your domain name for the auth service here.
+    # Volumes mounted in the compose.yml
+    ssl_certificate /opt/bitnami/nginx/conf/auth-cert.pem;
+    ssl_certificate_key /opt/bitnami/nginx/conf/auth-key.pem;
 
-    # The /auth paths go to the authentication and authorization service.
-    location /auth/admin/ {
-        proxy_pass http://auth:8080/auth/admin/;
+    # Keycloak can set large cookies and headers
+    proxy_buffer_size 16k;
+    proxy_buffers 8 16k;
+
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header X-Forwarded-Host $host;
+    proxy_set_header Host $http_host;
+    proxy_set_header X-NginX-Proxy true;
+
+    # The authentication and authorization service.
+    location /admin/ {
+        proxy_pass http://auth:8080/admin/;
     }
 
-    location /auth/js/ {
-        proxy_pass http://auth:8080/auth/js/;
+    location /js/ {
+        proxy_pass http://auth:8080/js/;
     }
 
-    location /auth/realms/ {
-        proxy_pass http://auth:8080/auth/realms/;
+    location /realms/ {
+        proxy_pass http://auth:8080/realms/;
     }
 
-    location /auth/resources/ {
-        proxy_pass http://auth:8080/auth/resources/;
+    location /resources/ {
+        proxy_pass http://auth:8080/resources/;
     }
 
-    location /auth/robots.txt {
-        proxy_pass http://auth:8080/auth/robots.txt;
+    location /robots.txt {
+        proxy_pass http://auth:8080/robots.txt;
+    }
```

Comment:
+1 this matches up with the [Keycloak reverse proxy docs](https://www.keycloak.org/server/reverseproxy)!